crates/rpc/rpc/src/eth/core.rs

```diff
@@ -424,6 +431,18 @@ where
     pub const fn blocking_task_guard(&self) -> &BlockingTaskGuard {
         &self.blocking_task_guard
     }
+
+    /// Returns [`EventStream`] of new raw transactions.
+    #[inline]
+    pub fn subscribe_to_raw_transactions(&self) -> EventStream<Bytes> {
+        self.raw_tx_sender.new_listener()
+    }
+
+    /// Broadcasts raw transaction to all subscribers.
+    #[inline]
+    pub fn broadcast_raw_transaction(&self, raw_tx: Bytes) {
+        self.raw_tx_sender.notify(raw_tx)
+    }
```

Comment:
this should be private and only check if there's at least 1 subscriber

Reply:
making it private means `send_raw_transaction` can't modify `EthAPIInner` state for `raw_tx_sender `